README.md

```diff
@@ -29,15 +29,20 @@ Why?
 
 ## Usage
 
-To clone the repository and run this website on your local machine, [install Jekyll](https://jekyllrb.com/docs/installation/) for your OS and type the following commands:
+### Dependencies
 
-    $ sudo gem install jekyll bundler
+Install [asdf-vm](https://asdf-vm.github.io/asdf/#/core-manage-asdf-vm) for your OS and now using asdf-vm, install [ruby](https://github.com/asdf-vm/asdf-ruby).
+
+### Cloning and running
```

Comment:
I used `cloning and running ` because at that time, I didn't get the right word in my mind.
Can we have something more appropriate that describe this section better in two or three words ? IDK whether this will fit to this `Build and Developement`, if you have better words that can describe this section go with that. Because I am myself doubtful with my suggestion. But something diff. from `Cloning and Running`
Else everything perfect :+1:

Reply:
ok i will see :)